src/constants/constants.js

```diff
@@ -0,0 +1,53 @@
+export const MAX_NUMBER_PRECISION = 15;
+
+export const BUTTONS_NAMES = {
+  MULTIPLY: 'multiply',
+  DIVIDE: 'divide',
+  SQUARE: 'square',
+  SQRT: 'sqrt',
+  CLEAR: 'CLEAR',
+  CE: 'CE',
+  PI: 'pi',
+  EQUAL: 'equal',
+  POWER: 'power',
+  OPEN_PARENTHESIS: 'openParenthesis',
+  CLOSING_PARENTHESIS: 'closingParenthesis',
```

Comment:
OPEN / CLOSE, OPENING / CLOSING... not sure if OPEN / CLOSING is how it's used. Just a question.